handlers/main.yml

```diff
@@ -36,3 +28,22 @@
   when:
     - openhpc_slurm_service_started | bool
     - openhpc_slurm_control_host in play_hosts
```

Comment:
Maybe I am struggling to work out how this was triggered before, and if that changes with this change... this does feel quite messy.